Git push rejected

## Description
<!-- Describe your issue -->
https://github.com/elimu-ai/content-provider/actions/runs/14724356518/job/41323844718#step:14:14
```sh
Run git push
To https://github.com/elimu-ai/content-provider
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/elimu-ai/content-provider'
hint: Updates were rejected because the remote contains work that you do not
hint: have locally. This is usually caused by another repository pushing to
hint: the same ref. If you want to integrate the remote changes, use
hint: 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
Error: Process completed with exit code 1.
```


## Why?
<!-- Explain _why_ this issue is needed -->


---

> elimu.ai's mission is to build innovative learning software that empowers out-of-school children to teach themselves basic reading📖, writing✍🏽 and math🔢 **within 6 months**.

--- a/.github/workflows/gradle-release.yml
+++ b/.github/workflows/gradle-release.yml
@@ -59,6 +59,11 @@ jobs:
           git add utils/build.gradle
           git commit -m "chore: prepare for next development iteration"
 
+      - name: Git Pull
+        run: |
+          git fetch
+          git pull
+
       - name: Git Push
         run: |
           git push

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -15,8 +15,8 @@ android {
         applicationId "ai.elimu.content_provider"
         minSdkVersion 26
         targetSdkVersion 35
-        versionCode 1002052
-        versionName "1.2.52-SNAPSHOT"
+        versionCode 1002053
+        versionName "1.2.53-SNAPSHOT"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         javaCompileOptions {
             annotationProcessorOptions {

--- a/utils/build.gradle
+++ b/utils/build.gradle
@@ -14,8 +14,8 @@ android {
     defaultConfig {
         minSdkVersion 26
         targetSdkVersion 35
-        versionCode 1002052
-        versionName "1.2.52-SNAPSHOT"
+        versionCode 1002053
+        versionName "1.2.53-SNAPSHOT"
     }
 
     buildTypes {

